packages/proxy/lib/http/response-middleware.ts

```diff
@@ -383,4 +414,5 @@ export default {
   MaybeRemoveSecurity,
   GzipBody,
   SendResponseBodyToClient,
+  MaybeStripDocumentDomainFeaturePolicy,
```

Comment:
These are evaluated in key-order, so move this to be right after `MaybePreventCaching`